src/bartender/web/api/job/views.py

```diff
@@ -193,3 +193,62 @@ def retrieve_job_stderr(
         Content of standard error.
     """
     return retrieve_job_files("stderr.txt", job_dir)
+
+
+@router.get(
+    "/{jobid}/directories",
+    response_model=DirectoryItem,
+    response_model_exclude_none=True,
+)
+async def retrieve_job_directories(
+    depth: PositiveInt = 1,
```

Comment:
technically should be `maxdepth`, no?

Reply:
indeed, the walk_dir also uses max_depth. Renamed it.